openMHA.cfg

```diff
@@ -9,9 +9,9 @@ mha.fragsize = 48
 mha.plugin_name = transducers
 
 mha.transducers.calib_in.peaklevel = [111.4 111.4]
-mha.transducers.calib_in.fir = [[0.05715807311106352 -0.09143269871454363 0.1279991931078954 -0.1229917176472585 0.03865469459901275 0.1208478993167953 -0.8820088913999001 -0.2804647683673479 0.01220373264309236 -0.1613181711981604 0.1851746406520001 -0.1149274499260817 0.04848940622236251 0.03877744359967165 -0.0533381761233464 0.06159030633073432];[0.05715807311106352 -0.09143269871454363 0.1279991931078954 -0.1229917176472585 0.03865469459901275 0.1208478993167953 -0.8820088913999001 -0.2804647683673479 0.01220373264309236 -0.1613181711981604 0.1851746406520001 -0.1149274499260817 0.04848940622236251 0.03877744359967165 -0.0533381761233464 0.06159030633073432]]
+mha.transducers.calib_in.fir = [[0.38830 -51.63502 -21.61600 2.13690 -5.86821 6.76826 -3.84954 2.51246];[0.38830 -51.63502 -21.61600 2.13690 -5.86821 6.76826 -3.84954 2.51246]]
```

Comment:
Probably these coefficients are for a samplerate of 44100 Hz.

Reply:
The changes should not change the frequency response too much, however 
`A = [[0.05715807311106352 -0.09143269871454363 0.1279991931078954 -0.1229917176472585 0.03865469459901275 0.1208478993167953 -0.8820088913999001 -0.2804647683673479 0.01220373264309236 -0.1613181711981604 0.1851746406520001 -0.1149274499260817 0.04848940622236251 0.03877744359967165 -0.0533381761233464 0.06159030633073432]];
B = [[0.38830 -51.63502 -21.61600 2.13690 -5.86821 6.76826 -3.84954 2.51246]];
plot(20.*log10(abs(fft(A,48000))));
hold on;
plot(20.*log10(abs(fft(B,48000))))`
shows a huge difference in the gains due to the old and new filter coefficients.

Reply:
Mit geänderten 48000 schlage ich
0.0017587   0.0800005  -0.0386776  -0.0132063   0.1627805   0.0112068  -0.1299747   0.0074928
vor

Reply:
Schau doch bitte mal ob du gleiche Ergebnisse bekommst für N=8, D=6. Sollte das nicht der Fall sein muss sich noch ein Fehler in meiner oder deine Berechnung befinden. Ich meine, dass ihr meine Skripts abgesegnet habt..

Reply:
Die neuen Koeffizienten sehen auf jefen Fall besser aus.
Ich kann aber grad nicht nachvollziehen warum nur wegen der Samplerate der Unterschied zu den Koeffizienten vorher so groß ist.
Trotzdem ist der Unterschied zwischen diesen Koeffizienten und den vorherigen noch erheblich (>15dB).
`A = [[0.05715807311106352 -0.09143269871454363 0.1279991931078954 -0.1229917176472585 0.03865469459901275 0.1208478993167953 -0.8820088913999001 -0.2804647683673479 0.01220373264309236 -0.1613181711981604 0.1851746406520001 -0.1149274499260817 0.04848940622236251 0.03877744359967165 -0.0533381761233464 0.06159030633073432]]; B = [[0.0017587 0.0800005 -0.0386776 -0.0132063 0.1627805 0.0112068 -0.1299747 0.0074928]]; plot(20.*log10(abs(fft(A,48000)))); hold on; plot(20.*log10(abs(fft(B,48000))))`

Bei mit funktioniert unter Octave das Script "Inversion_mic_parameters_plot" nicht.
Es fehlt der Pfad zu "tools".
Füge ich ihn manuell hinzu bekomem ich folgenden Fehler:

> error: vf_sig(1,_,_,_): but vf_sig has size 0x0
error: called from
    fftR at line 35 column 8
    average_imp at line 35 column 8
    Calculation_inversion_mic at line 18 column 15
    Inversion_mic_parameter_plots at line 20 column 39

Reply:
Der Fehler lag daran, dass die Pfade fest für Windowssysteme kodiert waren.
Ich habe die Anpassungen für eine Betriebssystemunabhängige Lösung gemacht und gepusht.
Trotzdem kommen bei mir andere Werte raus.

Ich habe die .mat files aus der Owncloud-Freigabe geladen
Kannst du mal schauen ob, wenn du die Dateien (sowohl Skripte als auch die Impulsantworten) frisch runterlädst noch dieselben Ergebnisse bekommst?

Bei mir kommt mit "Inversion_mic_parameters_plot" folgendes raus:
![example](https://user-images.githubusercontent.com/12151832/47990886-f8712800-e0e8-11e8-9a7c-cb145115ddf8.png)